test/integration/FileBackendEncodedSlashHandling.test.ts

```diff
@@ -53,10 +52,10 @@ describe('A server with a file backend storage', (): void => {
       body: 'abc',
     });
     expect(res.status).toBe(201);
-    expect(res.headers.get('location')).toBe(url);
+    expect(res.headers.get('location')).toBe(`${baseUrl}c1/c2/t1%2F`);
 
-    // The resource should not be accessible through ${baseUrl}/c1/c2/t1/.
-    const check1 = await fetch(`${baseUrl}/c1/c2/t1/}`, {
+    // The resource should not be accessible through ${baseUrl}c1/c2/t1/.
+    const check1 = await fetch(`${baseUrl}c1/c2/t1/}`, {
```

Comment:
Perhaps we should just use the `joinUrl` helper function throughout these tests to prevent accidental mistakes like this. Except perhaps for edge cases where we test double slashes (not sure how that library handles those).

Reply:
I'd just be explicit here with string concatenation, so we can more explicitly see what is happening.